lib/Dialect/AIEVec/Transforms/ConvertVectorToAIEVec.cpp

```diff
@@ -1427,10 +1427,9 @@ populateAIEVecV2TransformationPatterns(RewritePatternSet &patterns) {
   patterns.add<FoldAIEShiftAndBroadcast>(patterns.getContext());
 }
 
-static void
-populateAIEVecConvOpTransformationPatterns(RewritePatternSet &patterns,
-                                           unsigned shiftParam) {
-  patterns.add<FoldMulAddChainToConvOpPattern>(patterns.getContext(),
+static void populateAIEVecConvOpTransformationPatterns(
+    RewritePatternSet &patterns, AnalysisManager &am, unsigned shiftParam) {
+  patterns.add<FoldMulAddChainToConvOpPattern>(patterns.getContext(), am,
```

Comment:
The declaration of this function should be in `FoldMulAddChainToConvOp.h`, and its implementation in `FoldMulAddChainToConvOp.cpp`.